UnitsNet/CustomCode/Quantities/Area.extra.cs

```diff
@@ -47,16 +64,6 @@ public partial struct Area
         {
             return VolumeFlow.FromCubicMetersPerSecond(area.SquareMeters * speed.MetersPerSecond);
         }
-
-        public static Area FromCircleDiameter(Length diameter)
```

Comment:
Noticed that this was in a #!if WINDOWS_UWP block. No reason to bar them from WRC clients. Had to change the logic to avoid operator overloads. It's a bit more explicit now too.

Reply:
Nice find.